src/main/java/fashionable/simba/yanawaserver/matching/domain/Level.java

```diff
@@ -19,6 +21,10 @@ public Level(double level) {
         this.level = level;
     }
 
+    public Level() {
+
+    }
```

Comment:
```suggestion
    protected Level() {/*no-op*/}
```

기본 생성자를 외부에 공개할 필요는 없습니다.